client/client.go

```diff
@@ -204,3 +219,22 @@ func (c *Client) SetAccountSid(sid string) {
 func (c *Client) AccountSid() string {
 	return c.accountSid
 }
+
+func CheckTokenExpiry(tokenString string) bool {
```

Comment:
Maybe rename to `TokenExpired(tokenString string) bool` which makes clear that "true" means the token is expired? 

Just "checking" doesn't make clear whether we're checking true or checking false.